flight/Modules/Autotune/autotune.c

```diff
@@ -123,6 +123,8 @@ static void UpdateSystemIdent(const float *X, const float *noise,
 		relay.Noise[SYSTEMIDENT_NOISE_YAW]   = noise[2];
 	}
 	relay.Period = dT_s * 1000.0f;
+
+	relay.NumAfPredicts = predicts;
```

Comment:
I'm not sure what it stands for; it's the number of times af_predict is called, which does one step of the EKF.  I don't know what the af prefix in the module is.